src/main/java/model/figure/RightTriangle.java

```diff
@@ -0,0 +1,26 @@
+package model.figure;
+
+import model.Figure;
+
+public class RightTriangle extends Figure {
```

Comment:
The empty line after the class declaration is not necessary and should be removed.